Mathlib/Algebra/PUnitInstances.lean

```diff
@@ -0,0 +1,184 @@
+/-
+Copyright (c) 2019 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+
+! This file was ported from Lean 3 source module algebra.punit_instances
+! leanprover-community/mathlib commit 6cb77a8eaff0ddd100e87b1591c6d3ad319514ff
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Algebra.Module.Basic
+import Mathlib.Algebra.GCDMonoid.Basic
+import Mathlib.Algebra.GroupRingAction.Basic
+import Mathlib.GroupTheory.GroupAction.Defs
+import Mathlib.Order.CompleteBooleanAlgebra
+
+/-!
+# Instances on PUnit
+
+This file collects facts about algebraic structures on the one-element type, e.g. that it is a
+commutative ring.
+-/
+
+
+universe u
+
+namespace PUnit
+
+variable {R S : Type _} (x y : PUnit.{u + 1}) (s : Set PUnit.{u + 1})
+
+@[to_additive]
+instance commGroup: CommGroup PUnit where
+  mul := fun _ _ => PUnit.unit
+  one := PUnit.unit
+  inv := fun _ => PUnit.unit
+  div := fun _ _ => PUnit.unit
+  npow := fun _ _ => PUnit.unit
+  zpow := fun _ _ => PUnit.unit
+  mul_assoc := by intros; rfl
+  one_mul := by intros; rfl
+  mul_one := by intros; rfl
+  mul_left_inv := by intros; rfl
+  mul_comm := by intros; rfl
+
+@[simp, to_additive]
+theorem one_eq : (1 : PUnit) = star :=
+  rfl
+#align punit.one_eq PUnit.one_eq
```

Comment:
Missing the `to_additive`-izations of the aligns, but maybe you just haven't got to it yet.

Reply:
Will align all the `to_additive`s

Reply:
Seem to have missed a few, this one included.